package.json

```diff
@@ -36,7 +36,7 @@
     "generate:proxy:fims-models": "rimraf generated/fims && gen-api-models --api-spec api_fims.yaml --out-dir generated/fims",
     "generate:proxy:public-models": "rimraf generated/public  && gen-api-models --api-spec api_public.yaml --out-dir generated/public",
     "generate:api:io": "rimraf generated/io-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-app/v4.0.1/openapi/index.yaml --no-strict --out-dir generated/io-api --request-types --response-decoders --client",
-    "generate:api:io-messages": "rimraf generated/io-messages-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-app-messages/v3.17.0/openapi/index.yaml --no-strict --out-dir generated/io-messages-api --request-types --response-decoders --client",
+    "generate:api:io-messages": "rimraf generated/io-messages-api && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/io-functions-app-messages/ICC-81--add-PN-category-to-message-and-fallback/openapi/index.yaml --no-strict --out-dir generated/io-messages-api --request-types --response-decoders --client",
```

Comment:
Add to PR description 

"This PR depends on https://github.com/pagopa/io-functions-app-messages/pull/25" will help us track when this one will be ready to be released